### ISSUE TITLE
canvas api

### ISSUE THREAD

**Issue by chenglou** [reasonml-community/rnb#14]

**Comment by bsansouci** [comment]
I'm not sure I understand the intent behind having this. This would make rnb much more web specific, and potentially heavy on native. What about having a simple canvas API as a library that we can point people to?

**Comment by jaredly** [comment]
Hmmm I think rbn probably wants a "js namespace", that has things that only make sense on the web, but for which we really want a canonical implementation/binding

**Comment by chenglou** [comment]
Yep. Enough user land canvas bindings. Also, we can use conditional compilation to strip these away for native.
